checkov/common/bridgecrew/platform_integration.py

```diff
@@ -58,7 +58,7 @@ def __init__(self):
         self.timestamp = None
         self.scan_reports = []
         self.bc_api_url = os.getenv('BC_API_URL', "https://www.bridgecrew.cloud/api/v1")
-        self.bc_source = os.getenv('BC_SOURCE', 'cli')
```

Comment:
This is not needed anymore, because we use a singleton instance of this class. These used to be a (really bad) workaround for the 2-3 different paths for initializing this class.